src/pages/DownloadMonitor.vue

```diff
@@ -19,14 +19,28 @@
                         <div class="border-at-bottom pad pad--sides">
                             <div class="card is-shadowless">
                                 <p><strong>{{ downloadObject.initialMods.join(", ") }}</strong></p>
-                                <p v-if="downloadObject.progress < 100">Downloading: {{ downloadObject.modName }}</p>
-                                <p>{{Math.min(Math.floor(downloadObject.progress), 100)}}% complete</p>
-                                <Progress v-if="!downloadObject.failed"
-                                    :max='100'
-                                    :value='downloadObject.progress'
-                                    :className="['is-info']"
-                                />
-                                <Progress v-else-if="downloadObject.failed"
+
+                                <div v-if="downloadObject.downloadProgress < 100">
```

Comment:
Now the download monitor does the annoying thing where progress bar resets to zero when a next phase starts. (This isn't even my final form.)

I was wondering if showing two progress bars side by side would work in this view?